### DISCUSSION TITLE
Adding sub-items to the menu

### DISCUSSION THREAD

**Discussion by marta-coronado** [GMOD/jbrowse-components#2740] [Q&A] [answered]
Hi there,

I've been trying to follow the [developer guide](https://jbrowse.org/jb2/docs/developer_guide/#common-plugin-use-cases) to add new items to the menu without no luck. Any help or guidance for a non-developer person is much appreciated!

What I want to add is quite simple, is just a _Tutorial_ sub-item under the current _Help_ menu, as well as a _Download track data_ sub-item under the _Tools_ menu. When you click such items,they will redirect you to a plain html page (independent of JBrowse2, with a tutorial in html, and a directory index for downloading tracks). 

Is there an easy way for doing that, without the need of developing a plugin?
Thank you very much!

Cheers.

**Comment by marta-coronado** [comment, marked as answer]
I don't know if it's the correct way, but I managed to do it using the "e.rootModel.appendToMenu"!

**Reply by garrettjstevens** [reply]
That is indeed the suggested way of adding to the menu. You can see more about the various commands you can use to modify the menu here: https://jbrowse.org/jb2/docs/api_guide/#rootmodel-menu-api

**Reply by marta-coronado** [reply]
Thank you!
